.gitignore

```diff
@@ -129,6 +129,11 @@ ENV/
 # vim
 *.swp
 
+# Emacs
+*.~
+*#*#
+*.#*
```

Comment:
Does the following list cover all the needed files for you?

```
*~
\#*\#
/.emacs.desktop
/.emacs.desktop.lock
*.elc
auto-save-list
tramp
.\#*
```

Those are from https://github.com/github/gitignore/blob/master/Global/Emacs.gitignore

Reply:
I usually need those three entries I added but it's more comprehensive. Thanks!